docs/components/theme.js

```diff
@@ -0,0 +1,472 @@
+// Settings
```

Comment:
Will be moved to the `formidable-landers` repo with https://github.com/FormidableLabs/formidable-landers/issues/4